test/com/gfredericks/test/chuck/properties_test.cljc

```diff
@@ -0,0 +1,34 @@
+(ns com.gfredericks.test.chuck.properties-test
+  (:require [clojure.test.check :as tc]
+            [clojure.test.check.generators :as gen]
+            [clojure.test.check.clojure-test :as ct
+             #?(:clj :refer :cljs :refer-macros) [defspec]]
+            [com.gfredericks.test.chuck.properties :as prop'
+             #?@(:cljs [:include-macros true])]
+            [com.gfredericks.test.chuck.generators :as gen'
+             #?@(:cljs [:include-macros true])]
+            #?(:clj  [clojure.test :refer :all]
+               :cljs [cljs.test :refer-macros [deftest is]])))
+
+(deftest it-handles-exceptions-correctly
+  (is
+   (instance? #?(:clj  Throwable
+                 :cljs js/Error)
+              (:result
+               (tc/quick-check 100
+                 (prop'/for-all [x gen/int]
+                   #?(:clj  (/ 4 0)
+                      :cljs (js/Error. "Oops"))))))))
```

Comment:
Changed it to (in a9c6904):

```clojure
(checking "subs on any string or nil" 100
  [s (gen/one-of [gen/string (gen/return nil)])]
  ;; going for uncaught-error-not-in-assertion here
  (let [res (subs s 1 2)]
    (is res)))
```

But also did something similar in exception-handling-test and now I get `{:pass 1, :fail 1, :error 0}` instead of `{:pass 0, :fail 1, :error 0}` in the `exception-detection-test` in cljs. In clj works fine as before, returning `{:pass 0, :fail 1, :error 0}`.

Not sure why this is happening, but I don't want to fix it just by changing it to a different expression or something like that... I want to understand it first.

Any ideas?